drivers/classic/src/initialize_atmos.c

```diff
@@ -101,39 +100,44 @@ initialize_atmos(atmos_data_struct    *atmos,
     double                     delta_t_plus;
     int                        have_dewpt;
     int                        have_shortwave;
-    int                        hour_offset_int;
-    int                        tmp_starthour, tmp_endhour;
-    short                      local_startyear;
-    short                      local_startmonth;
-    short                      local_startday;
-    short                      local_starthour;
-    unsigned short             day_in_year, year, month, days_in_month;
+    int                        sec_offset_int;
```

Comment:
Remove the `_int` from the variable name. This is not a coding style that has been followed in VIC, so having it for one or two variables only does not help. Note there are variable naming conventions that people used to follow ("Hungarian"), in which every integer started with an `i`, every pointer with a `p`, but it is actually difficult to be consistent in this (point to an array of structures, etc).